SPECS/ruby/ruby.spec

```
@@ -128,6 +129,8 @@ Provides:       rubygem-base64 = %{base64_version}-%{release}
 Provides:       rubygem(base64) = %{base64_version}-%{release}
 Provides:       rubygem-benchmark = %{benchmark_version}-%{release}
 Provides:       rubygem(benchmark) = %{benchmark_version}-%{release}
+Provides:       rubygem-bigdecimal = %{bigdecimal_version}-%{release}
+Provides:       rubygem(bigdecimal) = %{bigdecimal_version}-%{release}
```

For the sake of consistency and to be one the safe side, let's do the same thing we did for `rubygem-bundler` below:
```suggestion
Provides:       rubygem(bigdecimal) = %{bigdecimal_version}-%{release}
Obsoletes:      rubygem-bigdecimal < %{bigdecimal_version}-%{release}
Obsoletes:      rubygem(bigdecimal) < %{bigdecimal_version}-%{release}
```